docs/tutorial/quickstart.md

```diff
@@ -182,7 +183,7 @@ Or using the [httpie][httpie], command line tool...
     }
 
 
-Or directly through the browser...
+Or directly through the browser, by going to http://127.0.0.1:8000/users/ 
```

Comment:
I'm ok with just this addition, leaving the other two changes as per @xordoquy's comments.